config/initializers/expose_migrations_path.rb

```diff
@@ -8,8 +8,8 @@ module SequelRails
   class Migrations
     class << self
       def migrations_dir
-        File.join(Gem::Specification.find_by_name('ontohub-models').gem_dir,
-                  'db/migrate')
+        root_path = Gem::Specification.find_by_name('ontohub-models').gem_dir
```

Comment:
sequel-rails now expects `migrations_dir` to return a `Pathname` instead of a `String`.